src/Disks/ObjectStorages/AzureBlobStorage/AzureObjectStorage.h

```diff
@@ -40,6 +48,10 @@ struct AzureObjectStorageSettings
     size_t max_single_read_retries = 3;
     size_t max_single_download_retries = 3;
     int list_object_keys_size = 1000;
+    size_t min_upload_part_size = 16 * 1024 * 1024;
+    size_t max_upload_part_size = 5ULL * 1024 * 1024 * 1024;
+    size_t max_part_number = 10000;
```

Comment:
okay, thanks. I have removed these settings and simplified calculatePartSize as you suggested

Reply:
Please remove `min_upload_part_size` and `max_part_number` because they are not used.
`max_upload_part_size` (5 GB) is too big, `DBMS_DEFAULT_BUFFER_SIZE` (1MB) should be enough.

Reply:
addressed it